tools/bookrunner/src/util.rs

```diff
@@ -203,7 +203,8 @@ pub fn add_codegen_job(litani: &mut Litani, test_props: &TestProps) {
 pub fn add_verification_job(litani: &mut Litani, test_props: &TestProps) {
     let exit_status = if test_props.fail_step == Some(FailStep::Verification) { 10 } else { 0 };
     let mut kani = Command::new("kani");
-    kani.arg(&test_props.path).args(&test_props.kani_args);
+    // Some of test_props.kani_args will contains `--cbmc-args` so we should always put that last
+    kani.arg(&test_props.path).args(&["--function", "main"]).args(&test_props.kani_args);
```

Comment:
Are we adding `--function main` to avoid having to annotate the bookrunner examples with `#[kani::proof]`? If so, can we add a comment?